uxarray/utils/helpers.py

```diff
@@ -0,0 +1,721 @@
+import numpy as np
+import xarray as xr
+from pathlib import PurePath
+from .get_quadratureDG import get_gauss_quadratureDG, get_tri_quadratureDG
+from numba import njit, config
+from uxarray.utils.constants import ENABLE_JIT_CACHE, ENABLE_JIT
+import math
+
+from uxarray.utils.constants import INT_DTYPE, INT_FILL_VALUE
+
+config.DISABLE_JIT = not ENABLE_JIT
+
+
+def parse_grid_type(dataset):
```

Comment:
This whole module appears to have been left over from a merge, it is not needed anymore and can be deleted. 

Please move any new functions in here into their appropriate module, which I would say is `grid/connectivity.py`